projects/packages/publicize/src/class-publicize-setup.php

```diff
@@ -97,6 +99,29 @@ public static function register_core_options() {
 		( new Jetpack_Social_Settings\Dismissed_Notices() )->register();
 	}
 
+	/**
+	 * Things to do on admin_init.
+	 */
+	public static function admin_init_actions() {
+		( new Jetpack_Social_Settings\Dismissed_Notices() )->register();
+		add_action( 'current_screen', array( self::class, 'add_filters_and_actions_for_screen' ), 5 );
```

Comment:
`current_screen` is only run in the admin, so perhaps we could add this action directly in `on_jetpack_feature_publicize_enabled` rather than in an admin_init hook.

Reply:
Done in e6e84d69d5fb97bd7dc2f9dd8dd89b29cc4b85f6.

Thanks